clients/cmd/fluentd/lib/fluent/plugin/out_loki.rb

```diff
@@ -162,7 +162,10 @@ def write(chunk)
 
         res_summary = "#{res.code} #{res.message} #{res.body}"
         log.warn "failed to write post to #{@uri} (#{res_summary})"
-        log.debug Yajl.dump(body)
+        # Perform JSON conversion only if required by log level
+        if log.level <= 1
```

Comment:
It appears that fluentd offers constants for these log levels:
https://github.com/fluent/fluentd/blob/master/lib/fluent/log.rb#L37
Can you use that please?

Reply:
Good point, it's much more readable this way. :)